crossdock/main.go

```diff
@@ -19,8 +19,8 @@ import (
 const (
 	behaviorEndToEnd = "endtoend"
 
-	envAgentHostPort = "JAEGER_AGENT_HOST_PORT"
-	envQueryHostPort = "JAEGER_QUERY_HOST_PORT"
+	envCollectorSamplingHostPort = "JAEGER_COLLECTOR_SAMPLING_HOST_PORT"
+	envQueryHostPort             = "JAEGER_QUERY_HOST_PORT"
 
 	envQueryHealthcheckHostPort     = "JAEGER_QUERY_HC_HOST_PORT"
 	envCollectorHealthcheckHostPort = "JAEGER_COLLECTOR_HC_HOST_PORT"
```

Comment:
this name is actually used in `crossdock/docker-compose.yml`. Where are the others above used?